llvm/lib/Transforms/Scalar/MemCpyOptimizer.cpp

```diff
@@ -1481,28 +1481,35 @@ bool MemCpyOptPass::performStackMoveOptzn(Instruction *Load, Instruction *Store,
     Worklist.push_back(AI);
     unsigned MaxUsesToExplore = getDefaultMaxUsesToExploreForCaptureTracking();
     Worklist.reserve(MaxUsesToExplore);
-    SmallSet<const Use *, 20> Visited;
+    SmallSet<Instruction *, 20> Visited;
```

Comment:
Why did this get switched from Use to Instruction?

Reply:
Ah, ... Sorry, that was style noises, I thought it'd be neat to handle Instruction type uniformly if it can be used, (although the const modifier is dropped...). I could revert this because of no rationale for improvements.